src/sentry/api/serializers/rest_framework/rule.py

```diff
@@ -156,5 +172,13 @@ def save(self, rule):
             rule.data["conditions"] = self.validated_data["conditions"]
         if self.validated_data.get("frequency"):
             rule.data["frequency"] = self.validated_data["frequency"]
+        if self.validated_data.get("owner"):
+            owner = self.validated_data["owner"]
+            if owner.type == User:
+                rule.user = owner
+                rule.team = None
+            elif owner.type == Team:
+                rule.team = owner
+                rule.user = None
```

Comment:
Should this be assigning to `rule.owner`?

Reply:
Your tests should cover that this serializer is actually assigning the owner as expected

Reply:
I was confused for a bit because I thought I was, and that this should have triggered an error. This serializer doesn't seem to have .save() called on it. It seems to be used only to validate the data, and then that data is extracted and passed to `project_rules.Creator` as a dict. I'll update this code just in case we ever use it though